frontend/src/js/components/settings/role-management/RoleManagement.tsx

```diff
@@ -109,7 +120,7 @@ export const RoleManagement = () => {
           <DocsTooltip id={DOCSTIPS.rbac.id} />
         </InfoHintContainer>
       </div>
-      {rolesInitialized ? <DetailsTable columns={columns} items={items} onItemClick={onEditRole} /> : <Loader show={true} />}
+      {rolesInitialized || initTimedOut ? <DetailsTable columns={columns} items={items} onItemClick={onEditRole} /> : <Loader show={true} />}
```

Comment:
can't we set `rolesInitialized` to `true` in `finally` block of the fetch instead? I would avoid `setTimeout` workarounds if possible